src/app/App.js

```diff
@@ -1,4 +1,24 @@
-export const App = ({options}) => {
-}
+export const App = ({ options }) => {
```

Comment:
ogólnie da się całość zamknąć w jednej funkcji bez potrzeby jej wywoływania przy ładowaniu pliku, na szybko zrobiłam coś takiego (jakoś dziwnie github formatuje, ale mam nadzieję, że skumasz ideę):

`  const buttonRanking = document.querySelector('.button--ranking');
  document.querySelector('.mode__hall').hidden = true;
  buttonRanking.addEventListener('click', switchBtn);
  function switchBtn() {
    buttonRanking.lastElementChild.innerHTML === 'Hall of fame'
      ? (buttonRanking.lastElementChild.innerHTML = 'Rules')
      : (buttonRanking.lastElementChild.innerHTML = 'Hall of fame');
    buttonRanking.firstElementChild.classList.toggle('fa-id-badge');
    document.querySelector('.mode__hall').hidden = !document.querySelector(
      '.mode__hall',
    ).hidden;
    document.querySelector('.mode__rules').hidden = !document.querySelector(
      '.mode__rules',
    ).hidden;
  }`

Reply:
to ładnie wyglądało, ale niestety nie działało poprawnie przełączanie ikon przy przycisku, jeśli się klikało kilka razy (tylko pierwsze przełączenie na widok hall of fame poprawnie zmieniało ikonę, a potem już nie było ikony). proponuję inny zapis, wjedzie jak tylko zamienię na tabelkę te wyniki.

Reply:
no teraz imo wygląda dużo lepiej :)